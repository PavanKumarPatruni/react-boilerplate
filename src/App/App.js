import React from 'react';

import Header from '../components/Header';

import './App.scss';

import Icon from '../assets/images/bg.png';

const App = () => {
  return (
    <div>
      <Header />
      <img src={Icon} alt="ICON" />
    </div>
  );
};

export default App;
