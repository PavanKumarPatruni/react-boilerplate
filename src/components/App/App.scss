$bg: '#dfdfdf';

body {
  background: $bg;
}
