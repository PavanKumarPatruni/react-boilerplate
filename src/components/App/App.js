import React from 'react';
import './App.scss';

const App = () => {
  return <div>React App</div>;
};

export default App;
