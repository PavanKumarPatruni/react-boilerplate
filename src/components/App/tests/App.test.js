import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App';

test('App Component Test', () => {
  const wrapper = renderer.create(<App />).toJSON();
  expect(wrapper).toMatchSnapshot();
});
