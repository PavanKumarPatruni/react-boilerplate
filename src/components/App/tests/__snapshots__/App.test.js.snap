// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Test 1`] = `
<div>
  React App
</div>
`;
